recipes/python-tsp/a280.tsp

```diff
@@ -0,0 +1,287 @@
+NAME : a280
```

Comment:
Can we use a file from the sdist? Something like `tests/data/att532.tsp`?

Reply:
Yes, I can look into it.

Reply:
There are no data files in the sdist:

```
moorepants@nandi:Downloads$ cd python_tsp-0.4.0/
moorepants@nandi:python_tsp-0.4.0$ tree
.
├── LICENSE
├── PKG-INFO
├── pyproject.toml
├── python_tsp
│   ├── distances
│   │   ├── data_processing.py
│   │   ├── euclidean_distance.py
│   │   ├── great_circle_distance.py
│   │   ├── __init__.py
│   │   ├── osrm_distance.py
│   │   └── tsplib_distance.py
│   ├── exact
│   │   ├── branch_and_bound
│   │   │   ├── __init__.py
│   │   │   ├── node.py
│   │   │   ├── priority_queue.py
│   │   │   └── solver.py
│   │   ├── brute_force.py
│   │   ├── dynamic_programming.py
│   │   └── __init__.py
│   ├── heuristics
│   │   ├── __init__.py
│   │   ├── local_search.py
│   │   ├── perturbation_schemes.py
│   │   └── simulated_annealing.py
│   ├── __init__.py
│   └── utils
│       ├── __init__.py
│       ├── permutation_distance.py
│       └── setup_initial_solution.py
└── README_pypi.rst
```